src/api/views.py

```diff
@@ -201,93 +230,24 @@ def convert(request):
-                            retval = rdftotagclass.onlineFunction([
-                                uploaded_file_path,
-                                folder+"/"+convertfile
-                                ])
-                            if (len(retval) > 0):
-                                warningoccurred = True
-                        elif (option2=="Spreadsheet"):
-                            rdftosprdclass = package.RdfToSpreadsheet
-                            retval = rdftosprdclass.onlineFunction([
-                                uploaded_file_path,
-                                folder+"/"+convertfile
-                                ])
-                            if (len(retval) > 0):
-                                warningoccurred = True
-                        elif (option2=="HTML"):
-                            rdftohtmlclass = package.RdfToHtml
-                            retval = rdftohtmlclass.onlineFunction([
-                                uploaded_file_path,
-                                folder+"/"+convertfile
-                                ])
-                            if (len(retval) > 0):
-                                warningoccurred = True
-                        else :
-                            message, returnstatus, httpstatus = convertError('400')
-                    elif (option1=="Spreadsheet"):
-                        print ("Verifing for Spreadsheet Document")
-                        if (option2=="Tag"):
-                            sprdtotagclass = package.SpreadsheetToTag
-                            retval = sprdtotagclass.onlineFunction([
-                                uploaded_file_path,
-                                folder+"/"+convertfile
-                                ])
-                            if (len(retval) > 0):
-                                warningoccurred = True
-                        elif (option2=="RDF"):
-                            sprdtordfclass = package.SpreadsheetToRDF
-                            retval = sprdtordfclass.onlineFunction([
-                                uploaded_file_path,
-                                folder+"/"+convertfile
-                                ])
-                            if (len(retval) > 0):
-                                warningoccurred = True
-                        else :
-                            message, returnstatus, httpstatus = convertError('400')
+                    fromFileFormat = serFileTypeEnum.valueOf(option1);
+                    toFileFormat = serFileTypeEnum.valueOf(option2);
+                    spdxConverter.convert(uploaded_file_path,
+                                folder+sep+convertfile, fromFileFormat, toFileFormat)
+                    retval = verifyclass.verify(folder+sep+convertfile, toFileFormat)
+                    if (len(retval) > 0):
+                        warningoccurred = True
                     if (warningoccurred == True ):
```

Comment:
We can change this to if `if (len(retval) > 0):` since warningoccurred will be True only then.

Reply:
Fixed in commit 981e0d5